### ISSUE TITLE
Determine replacement MTI cipher suite

### ISSUE THREAD

**Issue by ekr** [tlswg/tls13-spec#32]
Now that we are removing static RSA cipher suites, we need a new MTI cipher suite.

**Comment by ekr** [comment]
From Sean's mail

o Symmetric:
        MUST AES-GCM 128
        [SHOULD ChaCha20-Poly1305]

o Hash:
        MUST SHA-256

o Key Agreement: ECDH
        MUST P-256
        [SHOULD 25519]

o Signature:
        MUST ECDSA P-256
        MUST RSA

**Comment by dawud-tan** [comment]
Dear Mr. @ekr, currently I'm very interested in studying non-repudiation, then I found S-HTTP (RFC 2660) that offer non-repudiation security service. I also found that AS2 (RFC 4130) also offers this kind of service, but then I wonder why non-repudiation service is not becoming ubiquotus? Why AEAD replace HMAC, does hash function is too heavy, or hash is more appropriate for digital signature world? Thanks
